### PR TITLE
Docs: Make the grid values back from v4

### PR DESCRIPTION
**Pull request by louismaximepiton**
Backporting the values that were in [Grid options](https://boosted.orange.com/docs/4.6/layout/grid/#grid-options).

In [Available breakpoints](https://deploy-preview-1440--boosted.netlify.app/docs/5.2/layout/breakpoints/#available-breakpoints).
In [How they work table](https://deploy-preview-1440--boosted.netlify.app/docs/5.2/layout/containers/#how-they-work).
In [Grid options](https://deploy-preview-1440--boosted.netlify.app/docs/5.2/layout/grid/#grid-options).

### CODE REVIEW

**Review by isabellechanclou — APPROVED** [review]

File: site/content/docs/5.2/layout/containers.md [hunk 1]
```diff
@@ -24,7 +24,7 @@ See them in action and compare them in our [Grid example]({{< docsref "/examples
 Please note that we apply an extra-padding on our containers (see [our mixins](#mixins))
 
 {{< bs-table "table" >}}
```
Comment (julien-deramond) [inline site/content/docs/5.2/layout/containers.md:26]: Some values are not correct in this table.

Tested quickly with:
```html
<div class="container bg-dark" style="height: 40px">container</div>
<div class="container-sm bg-warning" style="height: 40px">container-sm</div>
<div class="container-md bg-warning" style="height: 40px">container-md</div>
<div class="container-lg bg-warning" style="height: 40px">container-lg</div>
<div class="container-xl bg-warning" style="height: 40px">container-xl</div>
<div class="container-xxl bg-warning" style="height: 40px">container-xxl</div>
<div class="container-fluid bg-warning" style="height: 40px">container-fluid</div>
```

Please check if the behavior is correct. If so, update the values in this table to reflect it.

File: site/content/docs/5.2/layout/grid.md [hunk 2]
```diff
@@ -97,12 +97,12 @@ As noted above, each of these breakpoints have their own container, unique class
     <tbody>
       <tr>
         <th class="text-nowrap" scope="row">Container <code class="fw-normal">max-width</code></th>
-        <td>320px</td>
-        <td>480px</td>
-        <td>768px</td>
-        <td>1024px</td>
-        <td>1280px</td>
-        <td>1440px</td>
+        <td>302px</td>
```
Comment (julien-deramond) [inline site/content/docs/5.2/layout/grid.md:100]: Are you sure those are the right values? It is mentioned that they must be the `max-width`.

**Review by julien-deramond — CHANGES_REQUESTED** [review]

File: site/content/docs/5.2/layout/breakpoints.md [hunk 3]
```diff
@@ -20,14 +20,14 @@ toc: true
...
 {{< bs-table "table" >}}
-| Breakpoint | Class infix | Dimensions |
-| --- | --- | --- |
-| Extra small | <em>None</em>  |&lt;480px |
-| Small | `sm` | &ge;480px |
-| Medium | `md` | &ge;768px |
-| Large | `lg` | &ge;1024px |
-| Extra large | `xl` | &ge;1280px |
-| Extra extra large | `xxl` | &ge;1440px |
+| Breakpoint | Class infix | Dimensions | Device |
```
[the remark this answers is not in the document]
Comment (louismaximepiton) [reply site/content/docs/5.2/layout/breakpoints.md:23]: Wasn't sure about this backport indeed, but wanted to give it a try (was already done by [Boosted on v4](https://boosted.orange.com/docs/4.6/layout/grid/#grid-options)). I'm reverting this for sure.

File: site/content/docs/5.2/layout/containers.md (hunk 1 above)
Comment (louismaximepiton) [reply site/content/docs/5.2/layout/containers.md:26]: I think that they are great 🤔

File: site/content/docs/5.2/layout/grid.md (hunk 2 above)
Comment (louismaximepiton) [reply site/content/docs/5.2/layout/grid.md:100]: You're right, it should be the same as mentioned in containers ! Changing them.

**Review by julien-deramond — APPROVED** [review]

**Pull request merged**